The test failure looks real
```
======================================================================
ERROR: test_format_both (corehq.apps.userreports.tests.test_columns.TestPercentageColumn)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/tests/test_columns.py", line 287, in test_format_both
    self.assertEqual('33% (1/3)', wrapped.format_fn()({'num': 1, 'denom': 3}))
AttributeError: 'PercentageColumn' object has no attribute 'format_fn'
```

--- a/corehq/apps/userreports/tests/test_columns.py
+++ b/corehq/apps/userreports/tests/test_columns.py
@@ -266,25 +266,25 @@ class TestPercentageColumn(SimpleTestCase):
         spec = self._test_spec()
         spec['format'] = 'percent'
         wrapped = ReportColumnFactory.from_spec(spec)
-        self.assertEqual('33%', wrapped.format_fn()({'num': 1, 'denom': 3}))
+        self.assertEqual('33%', wrapped.get_format_fn()({'num': 1, 'denom': 3}))
 
     def test_format_pct_denom_0(self):
         spec = self._test_spec()
         spec['format'] = 'percent'
         wrapped = ReportColumnFactory.from_spec(spec)
-        self.assertEqual('--', wrapped.format_fn()({'num': 1, 'denom': 0}))
+        self.assertEqual('--', wrapped.get_format_fn()({'num': 1, 'denom': 0}))
 
     def test_format_fraction(self):
         spec = self._test_spec()
         spec['format'] = 'fraction'
         wrapped = ReportColumnFactory.from_spec(spec)
-        self.assertEqual('1/3', wrapped.format_fn()({'num': 1, 'denom': 3}))
+        self.assertEqual('1/3', wrapped.get_format_fn()({'num': 1, 'denom': 3}))
 
     def test_format_both(self):
         spec = self._test_spec()
         spec['format'] = 'both'
         wrapped = ReportColumnFactory.from_spec(spec)
-        self.assertEqual('33% (1/3)', wrapped.format_fn()({'num': 1, 'denom': 3}))
+        self.assertEqual('33% (1/3)', wrapped.get_format_fn()({'num': 1, 'denom': 3}))
 
     def _test_spec(self):
         return {

